src/wallet/wallet.cpp

```diff
@@ -1899,12 +1899,8 @@ bool CWalletTx::RelayWalletTransaction(interfaces::Chain::Lock& locked_chain, CC
         /* GetDepthInMainChain already catches known conflicts. */
         if (InMempool() || AcceptToMemoryPool(locked_chain, maxTxFee, state)) {
             pwallet->WalletLogPrintf("Relaying wtx %s\n", GetHash().ToString());
```

Comment:
From discussion in commit `Remove use of AcceptToMemoryPool in wallet code` between you and @jnewbery: this `Relaying wtx` log message is displayed even if `!p2pEnabled()`. I'm fine with moving the log message down, or adding a TODO, or just ignoring this.

Reply:
Leave it. Tidy ups can come in a later PR.